spec/views/about/show.html.haml_spec.rb

```diff
@@ -3,6 +3,8 @@
 require 'rails_helper'
 
 describe 'about/show.html.haml', without_verify_partial_doubles: true do
+  let(:commit_hash) { '8925731c9869f55780644304e4420a1998e52607' }
```

Comment:
At some point I think it will be useful to have a spec that tests for the presence of the commit hash in the view, which is why this is in a `let`.  I have not written that spec yet, however, and I have no objection to moving the hash inline with the double definition.